### ISSUE TITLE
Transfer function consistently use either period or frequency

### ISSUE THREAD

**Issue by kkappler** [simpeg/aurora#94]
aurora.transfer_function.TTFZ.TTFZ seems to index its "tf" with frequency, but its inverse signal covariance "cov_ss_inv" with period.

Pick one and be consistent.

Although it is fine to offer a config that lets you use one or the other, but not both in the same data structure

**Comment by kkappler** [comment]
I pushed a commit to address this, but there are some tests and more work required here. 

My concern is the following:
Each decimation level will be organized to either be in 
 1. increasing frequency & decreasing period OR
 2. decreasing frequency & increasing period

However, the decimation levels themselves broadly decrease in frequency as you move through the levels.    So if we use increasing frequency,  the axes will be not smooth when you merge the decimation levels.

Using xarrays sortby method can help here, but we need to make sure that there are no assumptions baked in about the decimation level arrays being frequency or period based.

STFT is cast in terms of frequency, and frequency_bands list the bands in ascending order ( from low frequency to high frequency).
I tried modifying the core self.periods() in transfer_function base to return an inverted list of frequencies.  This may make things smooth, but there maybe some unintended consequences due to implicit assumptions about increasing frequency.

Running tests and will report back.
